prodigy.el

```diff
@@ -97,6 +97,14 @@ An example is restarting a service."
   :group 'prodigy
   :type 'number)
 
+(defcustom prodigy-dired-function 'dired
+  "Function to use for `dired' .
+If set to `dired' it will use the standard `dired' function. You
+could use any other `dired' like function. To use e.g. `ranger'
+set it to `deer'"
+  :group 'prodigy
+  :type 'symbol)
```

Comment:
This should be a function type instead of symbol, that way Emacs can validate it's a function and even give better completion.

For extra bonus points use `radio` and pre-fill the most common options (so dired and deer for example) and add a row for "custom function" See https://www.gnu.org/software/emacs/manual/html_node/elisp/Composite-Types.html#Composite-Types

Reply:
For an example you can look here: https://github.com/Fuco1/smartparens/blob/master/smartparens.el#L1369

Reply:
I didn't know about this, thanks